src/SFA.DAS.EmployerIncentives.Api.AcceptanceTests/Steps/SendBankDetailsRequiredEmailSteps.cs

```diff
@@ -0,0 +1,101 @@
+﻿using AutoFixture;
+using FluentAssertions;
+using SFA.DAS.EmployerIncentives.Api.Types;
+using SFA.DAS.Notifications.Messages.Commands;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using TechTalk.SpecFlow;
+
+namespace SFA.DAS.EmployerIncentives.Api.AcceptanceTests.Steps
+{
+    [Binding]
+    [Scope(Feature = "SendBankDetailsRequiredEmail")]
+    public class SendBankDetailsRequiredEmailSteps : StepsBase
+    {
+        private TestContext _testContext;
+        private Fixture _fixture;
+        private SendBankDetailsEmailRequest _request;
+        private string _url;
+        private string _storageDirectory;
+
+        public SendBankDetailsRequiredEmailSteps(TestContext testContext) : base(testContext)
+        {
+            _testContext = testContext;
+            _fixture = new Fixture();
+            _url = "/api/EmailCommand/bank-details-required";
+            var projectDirectory = Directory.GetCurrentDirectory().Substring(0, Directory.GetCurrentDirectory().IndexOf("bin"));
```

Comment:
Would be better to create a MessageBus binding which returns an object and then add it to the Context then it can be re-used in other tests to retrieve and assert on a message.  See the functions proj that did something similar for publishing instead of reading.